packages/react-i18n/src/i18n.ts

```diff
@@ -479,24 +482,44 @@ export class I18n {
   //
   // For other currencies, e.g. CHF and OMR, the "symbol" is the ISO currency code.
   // In those cases, we return the full currency code without stripping the country.
-  private getShortCurrencySymbol(currencyCode = this.defaultCurrency || '') {
-    const currency = currencyCode || this.defaultCurrency || '';
+  private getShortCurrencySymbol(
+    currency: string = this.defaultCurrency || '',
```

Comment:
Yea I modified my comment to be clearer. `currency` is `string` so removing `||''` would lead to a `string = string | undefined` scenario which would return a TS warning.

If `this.defaultCurrency` is `undefined`, a casting would return a string `"undefined"` which is not what we want. I'll consider this resolved then.